app/src/main/java/de/westnordost/streetcomplete/quests/accepts_cash/AddAcceptsCash.kt

```diff
@@ -34,6 +36,12 @@ class AddAcceptsCash(o: OverpassMapDataAndGeometryApi) : SimpleOverpassQuestType
           or tourism ~ ${tourism.joinToString("|")}
         )
         and name and !payment:cash and !payment:coins and !payment:notes
+        and (
+          tourism !~ attraction|gallery|museum 
```

Comment:
This expression matches any element that is not tourism attraction, gallery or museum. Maybe this is as you intended, but it is not easy to read. Instead, I'd strongly prefer if you did it like f.e. this:

```
or tourism ~ ${tourismWithImpliedFees.joinToString("|")}
or tourism ~ ${tourismWithoutImpliedFees.joinToString("|")} and fee = yes
```

Reply:
Yes, that, together with the following 3 lines was exactly what I was intending because the `and (` part is either true if *not* museum, gallery, attraction or if museum, gallery, attraction *and* fee.
Yet, if you prefer them joined from a list, I'll incorporate that later.
K